util/src/main/java/tc/oc/pgm/util/nms/NMSHacks.java

```diff
@@ -112,31 +111,19 @@ static void sendPacketToViewers(Entity entity, Object packet) {
         profile.getProperties().putAll(entry.getKey(), entry.getValue());
       }
     }
-
-    if (BukkitUtils.isSportPaper()) {
```

Comment:
Don't try to cheap out and use reflection for everything.

Direct access is **significantly** faster, and for tablist that's a must

Reply:
That's fair